### ISSUE TITLE
Keys used in pipes or other .ts files only are reported as missing

### ISSUE THREAD

**Issue by Moritz1409** [svoboda-rabstvo/ngx-translate-lint#147]
# Bug report

## Describe the bug

If a translation key is only used in pipes or .ts classes of the component (so complex logic is not included in the view directly), the linting treats the key as missing.

**ngx-translate-lint@1.11.0** works fine.
**ngx-translate-lint@1.20.4** shows error that keys don't exist in project.

## To Reproduce

- Add a basic pipe to the project and use it in any view:
```
@Pipe({
  name: 'somePipe'
})

@Injectable()
export class SomePipe implements PipeTransform {
  constructor(private translateService: TranslateService) {
  }

  transform(value): string {
    return this.translateService.instant('key1');
  }
}
```
- Add `key1` to the translation file.
- Execute `ngx-translate-lint`
- Linting fails with error: `Key: 'key1' doesn't exist in project'`

## Expected behavior

Linting should not treat keys as not-existing if they are used in pipes or other methods of `*.component.ts` files.

**Comment by Moritz1409** [comment]
This also works as before with `--deepSearch` being enabled. Thanks again!

**Comment by svoboda-rabstvo** [comment]
Hi @Moritz1409 thanks for quick answer, :) I try to resolve your issue without this flag in the future. 

Have a nice day

**Comment by julianpoy** [comment]
@svoboda-rabstvo `--deepSearch` enabled takes far longer to run compared to the older versions before this flag existed. I see a 3x increase in time in comparison to version 1.14.1.

Unfortunately without the `--deepSearch` flag enabled, I get the same issue as the original issue poster.

**Comment by itekaf** [comment]
@julianpoy Yes, you're absolutely right. I'm working on it right now :) When I have update I will write here
